Change logs are good (#50)
ietf-gitwg/using-github: draft-ietf-git-using-github.md

tool has been used effectively in several Working Groups, though it requires
server infrastructure.

Additionally, clear reporting about the changes that were included in each

> revision of an Internet-Draft helps ensure that contributors are able to

Reviewer: ```suggestion
revision of an Internet-Draft helps ensure that contributors can
```
